src/components/seo.js

```diff
@@ -20,69 +19,45 @@ const SEO = (
           siteMetadata {
             title
             description
+            siteUrl
           }
         }
       }
     `,
   );
 
-  const metaDescription = description || site.siteMetadata.description;
+  const {title: siteTitle, description: siteDescription, siteUrl} = site.siteMetadata;
+  const metaDescription = description || siteDescription;
+  const absoluteOgImage = ogImage?.startsWith('http') ? ogImage : `${siteUrl}${ogImage}`;
 
   return (
-    <Helmet
-      htmlAttributes={{
-        lang,
-      }}
-      title={title}
-      titleTemplate={`%s | ${site.siteMetadata.title}`}
-      meta={[
-        {
-          name: `description`,
-          content: metaDescription,
-        },
-        {
-          property: `og:title`,
-          content: title,
-        },
-        {
-          property: `og:description`,
-          content: metaDescription,
-        },
-        {
-          property: `og:url`,
-          content: site.siteMetadata.siteUrl,
-        },
-        {
-          property: `og:type`,
-          content: `website`,
-        },
-        {
-          property: `og:image`,
-          content: `${site.siteMetadata.siteUrl}${ogImage}`,
-        },
-        {
-          property: `og:image:width`,
-          content: ogImageWidth,
-        },
-        {
-          property: `og:image:height`,
-          content: ogImageHeight,
-        },
-        {
-          name: `twitter:title`,
-          content: title,
-        },
-        {
-          name: `twitter:description`,
-          content: metaDescription,
-        },
-      ].concat(meta)}
-    />
+    <>
+      <title>{title} | {siteTitle}</title>
+      <link rel="canonical" href={siteUrl} />
+      <meta name="description" content={metaDescription} />
+      <meta property="og:title" content={title} />
+      <meta property="og:description" content={metaDescription} />
+      <meta property="og:url" content={siteUrl} />
```

Comment:
This always seems to be the root url and does not include the path. For instance if you are on https://dvp-landing-staging.web.app/track-record/ this is still https://dvp-landing-staging.web.app/. Which I think will mean we have incorrect shares. 

I think the change is in the query to `url` rather then `siteUrl`.